FirebaseStorageSwift/Sources/SwiftApiExtension.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseStorage
+
+/// getResultCallback generates a closure that returns a Result type from a closure that returns an optional type and Error.
```

Comment:
Please fix line length. go/swift-style calls for 100 cols.

Reply:
![Screen Shot 2020-03-30 at 4 21 56 PM](https://user-images.githubusercontent.com/10249104/77971097-b7ac1d00-72a2-11ea-9c02-8a963e2862bc.png)

For the scheme you're using, click one of the elements under Source Editor, hit cmd-a to select all, then select the font you want to have for all elements. For myself, I find color helpful, but difference in font unhelpful so I set them all the same.

Reply:
Thanks! I made the more minimal change of only changing Documentation Markup to SF Mono Regular 
![Screen Shot 2020-03-30 at 4 39 45 PM](https://user-images.githubusercontent.com/73870/77972044-31dda100-72a5-11ea-8977-cb359d3d2cfa.png)